.gitignore

```diff
@@ -4,3 +4,6 @@ docs
 .nyc_output
 coverage
 npm-debug.log
+
+# Prevent adding test fixture package-lock files from npm@5 
+test/**/package-lock.json
```

Comment:
I'd say just ignore all `package-lock.json` files.

Reply:
Just curious, is the reason why `electron-forge` isn't using `package-lock.json` or `yarn.lock` to control dependency upgrades because consumer won't use them anyway (so by not having them we should detect potential breaks earlier)?

Reply:
We're going to [require Yarn (well, technically bolt)](https://github.com/electron-userland/electron-forge/pull/433) in v6.